runtime/proto/service/log_replication.proto

```diff
@@ -52,4 +52,23 @@ enum LogReplicationEntryType {
   SNAPSHOT_REPLICATED = 4;
   SNAPSHOT_END = 5;
   SNAPSHOT_TRANSFER_COMPLETE = 6;
+}
+
+message LogReplicationSession {
+  string sinkClusterId = 1;
+  string sourceClusterId = 2;
```

Comment:
can we please use proto convention of _ ?

Reply:
sure, don't see why not.